src/groupchatdlg.cpp

```diff
@@ -1489,7 +1489,7 @@ void GCMainDlg::sendNewTopic(const QMap<LanguageManager::LangId, QString> &topic
     emit aSend(m);
 }
 
-void GCMainDlg::doShowInfo()
+void GCMainDlg::doShowMucInfos()
```

Comment:
revert this

Reply:
It is for MUC Room and there are informations, not only one.